_gettingstarted/4-python-highcharts-core.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: article
+title: Highcharts-core
+description: A comprehensive guide on how to work with Highchats-core, a Python library for creating dynamic charts.
+tags: [highcharts-core, charts, python, pip]
+image: https://learn.piptrends.com/assets/images/getting-started/getting-started-python-highcharts-cover.png
+cover: [/assets/images/getting-started/Python-Highcharts-Icon]
+article_header:
+  type: overlay
+  theme: dark
+  background_image:
+  gradient: 'linear-gradient(135deg, #82d2a6, #8087e8)'
```

Comment:
There needs to be a tab space here
gradient is supposed to be under background_image
![image](https://github.com/tankala/learn-pip-trends/assets/67141217/2532006c-455e-4d89-aa9c-d9b53fc98902)